.github/workflows/nightly_tests.yaml

```diff
@@ -229,8 +229,8 @@ jobs:
             - name: Install uv
               uses: astral-sh/setup-uv@v7
             - name: Run slint compiler via uxv
-              working-directory: api/python/compiler
+              working-directory: api/python/
               env:
                 SLINT_COMPILER_VERSION: nightly
               # TODO: generate python code when the python generator lands
-              run: uvx --with-editable=. slint-compiler ../../../demos/printerdemo/ui/printerdemo.slint
+              run: uvx --with-editable=. slint-compiler ../../demos/printerdemo/ui/printerdemo.slint
```

Comment:
With this being run in the `api/python` directory (where there's no `pyproject.toml`, I think this test will basically download the latest release from pypi.

If the compiler builds (which we test elsewhere), then I think we can rely on Maturin's packaging to work.
I think this entire test can be removed.